cmd/zpool/zpool_main.c

```diff
@@ -5864,8 +5884,17 @@ print_scan_status(pool_scan_stat_t *ps)
 	 * Scan is in progress.
 	 */
 	if (ps->pss_func == POOL_SCAN_SCRUB) {
-		(void) printf(gettext("scrub in progress since %s"),
-		    ctime(&start));
+		if (pause == 0) {
+			(void) printf(gettext("scrub in progress since %s"),
+			    ctime(&start));
+		} else {
+			size_t len = strlen(ctime(&pause));
+			char cbuf[len];
+			struct tm *p = localtime(&pause);
+			(void) strftime(cbuf, len, "%a %b %e %T %Y", p);
```

Comment:
Using `strlen(ctime())` as a way to calculate the maximum buffer size for `strftime()` is a little dangerous since the required buffer size depends on the pattern format specifier.  It would be simpler to safer to declare this buffer as say 32 bytes.